packages/app/src/debug/DebugSpec.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div
+    data-cy="debug-spec-col"
+    class="grid flex flex-col px-24px gap-24px self-stretch"
+  >
+    <div
+      data-cy="debug-spec-item"
+      class="w-full overflow-hidden flex flex-col items-start box-border border-t-1px border-x-1px rounded"
+    >
+      <div
+        data-cy="debug-spec-header"
+        class="w-full flex flex-row items-center py-12px bg-gray-50 border-b-1px border-b-gray-100 rounded-t"
+      >
+        <div
+          data-cy="spec-contents"
+          class="w-full flex flex-row px-16px items-center"
+        >
+          <div
+            data-cy="spec-path"
+            class="flex-grow non-italic text-base truncate"
+          >
+            <span
+              class="font-normal text-gray-600"
+            >
+              {{ specData.path }}
+            </span>
+            <span
+              class="font-medium text-gray-900"
+            >
+              {{ specData.fileName }}
+            </span>
+            <span
+              class="font-normal text-gray-600"
+            >
+              {{ specData.extension }}
+            </span>
+          </div>
+          <div
+            class="ml-10px"
+          >
+            <Button
```

Comment:
FYI - always a good practice to specify `type="button"` so it's obvious this isn't intended as a `type="submit"` form submission button. That way you don't accidentally end up submitting a form if this button happens to nested within one